cfpb/cfgov-refresh
# Move Blog into Wagtail

Short description explaining the high-level reason for the pull request

## Additions

- LegacyBlogPage
 - model and migration scripts

## Removals

- Sheer blog

## Changes

- some minor inconsequential stuff

## Testing

- Create a `blog` browse filterable page under about us with a filter
- Run `./cfgov/manage.py import_data posts legacyblogpage --parent blog -u user -p pass`
- Go to `/blog/` and see it in all its glory.

## Review

- @kave 
- @richaagarwal 
- @rosskarchner 

## Screenshots
![screen shot 2016-04-15 at 8 53 11 am](https://cloud.githubusercontent.com/assets/1412978/14561895/88cc3f94-02e7-11e6-9c8f-76d227d53c27.png)


## Todos

- The new BlogPage class needs to be made
- Still need to set up an RSS feed
- Need to convert newsroom into wagtail
